initial start on assistant docs (#201)
beaverbuilder/documentation: docs/assistant/cloud/collections.md

In the image below, the color `#ede2f1` is filed under two collections; **Pastel Colors** and **Pink Colors**.

![Organizing colors](/img/assistant/cloud--libraries--organize-assets--1.jpg)

## Create a Collection from the Assistant Pro Cloud

To create a library collection from the Assistant Pro Cloud:

1. Select the library you’d like to add a collection.
2. Select the **Collections** tab
3. Type a name for your collection
4. Click the **Add Collection** button

![Create collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--2.jpg)

## Create a Library Collection from the Assistant Plugin

To create a library collection from the Assistant Plugin:

1. Click on the library you’d like to add a collection.
2. Click the **Settings** icon.
3. Type a name for your collection
4. Click the **Add Collection** button

![Create collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--3.jpg)

> ## Assign Collections to a creative asset from Assistant Pro Cloud

Reviewer: ```suggestion
## Assign Collections to a creative asset from Assistant Pro Cloud or the Assistant plugin
```
As far as I can see, the procedure is identical so you can combine this section with the next one.

Author: I'd prefer to keep them separated.

To add collections to a creative asset:

> 1. Select the asset.

Reviewer: ```suggestion
1. Click on an asset to open its settings.
```

> 2. Enter collection names into the sidebar dialogue field. 

Reviewer: ```suggestion
2. In the **Collections** field, start typing a collection name. Collection names that you've already used in that library will appear as suggestions.
```

![Assign assets to collections from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--4.jpg)

> ## Assign Collections to a creative asset from the Assistant Plugin

Reviewer: If you agree this procedure is identical to the previous one, you can delete lines 48-53.

To add collections to a creative asset:

1. Select the asset.
2. Enter collection names into the sidebar dialogue field.

:::tip
If you enter a collection name that is unassigned, Assistant will create a new collection with that name. This functionality is available in both the Assistant plugin and Assistant Pro Cloud.
:::

![Assign assets to collections from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--5.jpg)

## Sort by Collection from the Assistant Pro Cloud

> ​​By default libraries are sorted by file type. Once you have added collections, you can also sort by collection.

Reviewer: ```suggestion
​​By default, library assets are sorted by file type. Once you have added collections, you can also sort by collection.
```

> View a list of a library’s collections by clicking the collections tab.

Reviewer: ```suggestion
To view a library’s assets grouped by collection, open a library and click the **Type** dropdown at the top of the page, then choose **Collection**.  
```

> ![Sort by collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--6.jpg)

Reviewer: ```suggestion
![Sort assets by collection from Assistant Pro Cloud](/img/assistant/cloud--libraries--organize-assets--6.jpg)
```

## Sort by Collection from the Assistant Plugin

> To sort a library by collection from the Assistant Plugin:

Reviewer: ```suggestion
To sort a library's assets by collection from the Assistant Plugin:
```

> * Select **View by Collection** from the dropdown menu.

Reviewer: ```suggestion
* Click the **View by** dropdown and choose **Collection**.
```

> ![Sort by collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--7.jpg)

Reviewer: ```suggestion
![Sort assets by collection from Assistant plugin](/img/assistant/cloud--libraries--organize-assets--7.jpg)
```
